content/id/docs/cluster-administration/proxies.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Berbagai Proxy di Kubernetes
+content_template: templates/concept
+weight: 90
+---
+
+{{% capture overview %}}
+Laman ini menjelaskan berbagai <i>proxy</i> yang ada di dalam Kubernetes.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Berbagai Jenis <i>Proxy</i>
+
+Ada beberapa jenis <i>proxy</i> yang akan kamu temui saat menggunakan Kubernetes:
+
+1.  [kubectl proxy](/docs/tasks/access-application-cluster/access-cluster/#directly-accessing-the-rest-api):
+
+    - dijalankan pada <i>desktop</i> pengguna atau di dalam sebuah pod
```

Comment:
if we're using an API object should we use capital as Pod or pod?

Reply:
true, Pod is better